src/views/DidUpgradeExplainer/DidUpgradeExplainer.tsx

```diff
@@ -26,12 +37,35 @@ export function DidUpgradeExplainer({ identity }: Props): JSX.Element | null {
 
   const wasOnChainDidDeleted = useIsOnChainDidDeleted(did);
 
+  const { total: kiltCosts, insufficientKilt } = useKiltCosts(address, did);
+
+  const [paymentMethod, setPaymentMethod] = useState<'kilt' | 'euro'>('kilt');
```

Comment:
Having this enumeration in two places already, would make sense to give it a name